jussi/validators.py

```diff
@@ -221,17 +221,20 @@ def is_get_block_header_request(
     try:
         return jsonrpc_request.urn.method == 'get_block_header'
     except Exception as e:
-        logger.debug('is_get_block_request errored: %s', e,
+        logger.warning('is_get_block_request errored: %s', e,
                      extra=jsonrpc_request.log_extra())
         return False
 
 
 def is_get_dynamic_global_properties_request(
         jsonrpc_request: SingleJsonRpcRequest = None) -> bool:
     try:
+        if not hasattr(jsonrpc_request, 'urn'):
```

Comment:
This check isn't necessary here, but the error you're trying to fix is a good one to look at. This function is called in one place: https://github.com/steemit/jussi/blob/master/jussi/middlewares/update_block_num.py#L22
The error happens when it is passed a jsonrpc batch request. Our fix should be either call it for each member of a batch request, or don't call it at all for batch requests. I vote for the latter, as I think it would be rare for this call to be included in a batch request. What do you think?

If you agree with me about the batch requests, then I think we should change this line: https://github.com/steemit/jussi/blob/master/jussi/middlewares/update_block_num.py#L17

I think it should be `if request.method != 'POST' or isinstance(request.json, list):`